### PR TITLE
KEYCLOAK-12845 Update repository names

### PR DESCRIPTION
**Pull request by hmlnarik**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: docs/templates/openj9/sso74-openj9-https.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@
 :toc-placement!:
 :toclevels: 5
 
-An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-openj9-7-openshift-image/docs.
+An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.
```
Comment (iankko) [inline docs/templates/openj9/sso74-openj9-https.adoc]: Above URL returns HTTP 404 (not found). Maybe replace with `https://github.com/jboss-container-images/redhat-sso-7-openshift-image/tree/sso74-dev/docs` instead?

File: docs/templates/openj9/sso74-openj9-postgresql-persistent.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@
 :toc-placement!:
 :toclevels: 5
 
-An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-openj9-7-openshift-image/docs.
+An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.
```
Comment (iankko) [inline docs/templates/openj9/sso74-openj9-postgresql-persistent.adoc]: Same as above, HTTP 404

File: docs/templates/openj9/sso74-openj9-postgresql.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@
 :toc-placement!:
 :toclevels: 5
 
-An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-openj9-7-openshift-image/docs.
+An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.
```
Comment (iankko) [inline docs/templates/openj9/sso74-openj9-postgresql.adoc]: Another instance

File: docs/templates/openj9/sso74-openj9-x509-https.adoc [hunk 4]
```diff
@@ -9,7 +9,7 @@
 :toc-placement!:
 :toclevels: 5
 
-An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-openj9-7-openshift-image/docs.
+An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.
```
Comment (iankko) [inline docs/templates/openj9/sso74-openj9-x509-https.adoc]: Here too

File: docs/templates/openj9/sso74-openj9-x509-postgresql-persistent.adoc [hunk 5]
```diff
@@ -9,7 +9,7 @@
 :toc-placement!:
 :toclevels: 5
 
-An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-openj9-7-openshift-image/docs.
+An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.
```
Comment (iankko) [inline docs/templates/openj9/sso74-openj9-x509-postgresql-persistent.adoc]: Hopefully final

**Review by iankko — CHANGES_REQUESTED** [review]
The templates base URL needs update yet IMHO (returns HTTP 404 currently)

The rest /lgtm

File: templates/update.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python3
```
Comment (iankko) [inline templates/update.py]: @hmlnarik Why is this script needed? Maybe to automate the indentation to four spaces for all templates (yet before it's passed to other scripts)?

If that's the case could it be updated to take the file to operate at as CLI arg?

Comment (hmlnarik) [reply templates/update.py]: Thanks @iankko , omission on my part, sorry!

Comment (iankko) [reply templates/update.py]: Ok, thanks! All clear now

**Review by iankko — APPROVED** [review]
/lgtm

**Pull request merged**